### DISCUSSION TITLE
Maintenance Mode possible?

### DISCUSSION THREAD

**Discussion by SOLTEK-007** [GetPublii/Publii#1512] [Help]
Is there a way to switch to a temporary maintenance mode, bringing the site offline and only showing a page that says the site is temporarily offline for maintenance? Or if not, is there a way to temporarily password protect the site so that no one can access it until I'm ready to unlock it?

**Comment by dziudek** [comment]
As the website is static it is impossible - you must to do it through your hosting configuration - e.g. on Apache you can use Basic Auth in .htaccess

**Comment by gpsblues** [comment]
Hi,

you can try with this unofficial plugin https://plugliin.eu.org/maintenance-mode/

If I remember correctly it displays a custom message via JS
